akka-stream/src/main/scala/akka/stream/scaladsl/FlexiMerge.scala

```diff
@@ -233,4 +236,10 @@ abstract class FlexiMerge[Out, S <: Shape](val shape: S, attributes: OperationAt
     case Some(n) ⇒ n
     case None    ⇒ super.toString
   }
+
+  // FIXME what to do about this?
```

Comment:
ok, fixing that, and also noticed that the attributes from the constructor is not passed to the module, fixing that also